### ISSUE TITLE
60 images loaded but only 14 images is shown

### ISSUE THREAD

**Issue by khaukheng** [eKoopmans/html2pdf.js#245]
![image](https://user-images.githubusercontent.com/17702727/62003189-cd7ea680-b145-11e9-834c-0096fe4e0cd0.png)

```
 let data = window.document.getElementById('contentToConvert');
      const opt = {
        margin:       10,
        filename:     'Inspection-Report-'+id+'.pdf',

        html2canvas:  { scale: 2 ,
                        letterRendering: true,
                        useCORS: true},
        pagebreak: { 
          mode: ['css', 'legacy'],
          avoid: ['collapsible','.avoid-page-break']
        },
        jsPDF:        { unit: 'mm', format: 'a2', orientation: 'portrait' },
      };

      await html2pdf().from(data).set(opt).outputPdf('arraybuffer')
      .then((result)=>{
        this.zipReport(result);
      }
```
Issue: The resulting pdf is 13 pages long, but the there should be 60 images shown in the pdf but only the first 13 images is shown and the rest is empty plus the content starting from image number 14 is empty except the <textarea> tag and the numbering. 
Please help @eKoopmans

**Comment by MigueBlogs** [comment]
How many pages have your PDF created? i have the same issue always after the page number 4.

**Comment by khaukheng** [comment]
> How many pages have your PDF created? i have the same issue always after the page number 4.

I've 13 pages of pdf created, the above attached image is page number 4 when it started to have empty image. However if I reduce the html2pdf scale to 1,it can show up to 30 photo, and if I reduce scale to 0.7, it can show up to 52 photo. I am suspecting it's because of the maximum pixel allowed in a single canvas.

**Comment by MigueBlogs** [comment]
YES! i have the same issue > https://github.com/eKoopmans/html2pdf.js/issues/247  detected, i have the same problem when i put the ` html2canvas: { scale: 5 },` the problem is presented on the page 4.

**Comment by gillespieza** [comment]
It's because you are hitting the maximum number of pixels allowed per canvas. Try splitting into multiple elements and chaining them into 1 pdf.
